readium/shared/src/main/java/org/readium/r2/shared/fetcher/Resource.kt

```diff
@@ -219,70 +219,18 @@ abstract class ProxyResource(protected val resource: Resource) : Resource {
 
 }
 
-/**
- * Caches the members of [resource] on first access, to optimize subsequent accesses.
- *
- * This can be useful when reading [resource] is expensive.
- *
- * Warning: bytes are read and cached entirely the first time, even if only a [range] is requested.
- * So this is not appropriate for large resources.
- */
-@Deprecated("If you were caching a TransformingResource, build it with cacheBytes set to true." +
```

Comment:
I think it's fine to remove this now. The `@Deprecated` was emitting an error so nobody should be using it by now.